### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -7864,6 +7864,8 @@ msgid ""
 "broken link from %7s %s\n"
 "              to %7s %s"
 msgstr ""
+"пошеоджене посилання з %7s %s\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"пошкоджене посилання з %7s %s\n"
```

File: po/uk.po [hunk 2]
```diff
@@ -8117,31 +8123,31 @@ msgstr "неприпустимий параметр: очікувався sha1,
...
 #, c-format
 msgid "value of '%s' not bool or int: %d"
-msgstr ""
+msgstr "значення \"%s\" не є bool або int: %d"
 
 #: builtin/fsmonitor--daemon.c:106
 #, c-format
 msgid "fsmonitor-daemon is watching '%s'\n"
-msgstr ""
+msgstr "fsmonitor-daemon стежить за '%s'\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "fsmonitor-daemon стежить за \"%s\"\n"
```

File: po/uk.po [hunk 3]
```diff
@@ -8345,38 +8365,39 @@ msgstr "не вдалося виконати попереднє отриманн
...
 #: builtin/gc.c:1198
 msgid "'git multi-pack-index repack' failed"
-msgstr ""
+msgstr "\"git multi-pack-index repack\" завершився невдало"
 
 #: builtin/gc.c:1207
 msgid ""
 "skipping incremental-repack task because core.multiPackIndex is disabled"
 msgstr ""
+"пропуск incremental-repack завдання, оскільки core.multiPackIndex вимкнено."
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"пропуск incremental-repack завдання, оскільки core.multiPackIndex вимкнено"
```

File: po/uk.po [hunk 4]
```diff
@@ -8345,38 +8365,39 @@ msgstr "не вдалося виконати попереднє отриманн
...
 
 #: builtin/gc.c:1311
 #, c-format
 msgid "lock file '%s' exists, skipping maintenance"
-msgstr ""
+msgstr "файл блокування \"%s\" існує, пропускаємо обслуговування"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "файл блокування \"%s\" існує, пропуск обслуговування"
```

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 👍

File: po/uk.po (hunk 3 above)
Comment (arkid15r) [reply po/uk.po]: 🔍

**Review by arkid15r — COMMENTED** [review]
Nice review!

**Pull request merged**